senders/telegram/send.go

```diff
@@ -22,6 +21,7 @@ const (
 	photoCaptionMaxCharacters     = 1024
 	messageMaxCharacters          = 4096
 	additionalInfoCharactersCount = 400
+	maxDescriptionSize            = 450
```

Comment:
But why 450?

You have actual `maxChars` value (1024 or 4096) in `buildMessage`. You have full information on `events` in `buildMessage`. Let's say `maxChars` is 4096 and you have only one event. Is it really necessary to cut description to just 450 chars?

You can refer to https://github.com/moira-alert/moira/blob/d87d8757234c5fe688d8388751fd4b53fdd4b22e/senders/slack/slack.go#L76 and reuse code to implement this better.